EU4toV2/Source/EU4World/Army/EU4Regiment.h

```diff
@@ -1,71 +1,64 @@
 #ifndef EU4_REGIMENT_H
 #define EU4_REGIMENT_H
-
+#include "ID.h"
 #include <map>
-#include "../ID.h"
 
 namespace EU4
 {
-	enum class REGIMENTCATEGORY 
-	{
-		// ground units
-		infantry,
-		cavalry,
-		artillery,
-		// navy units
-		heavy_ship,
-		light_ship,
-		galley,
-		transport,
-		// end of enum
-		num_reg_categories
-	};
+enum class REGIMENTCATEGORY
```

Comment:
I'm not really sure there's an advantage to using an enum here instead of just storing the original string (yes, I know, this is almost certainly something I originally wrote. I know disagree with my past self)

So maybe just rip these all out and store the string? There's a switch statement you'd have to rewrite as an if/else, but I don't see a problem with that.